styles/styles.css

```diff
@@ -0,0 +1,199 @@
+@import url("./fonts.css");
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.container {
+  margin-bottom: 120px;
+  padding: 120px 104px 122px;
+  width: 1520px;
```

Comment:
Where did you get this width?
Btw it could be flexible.

Reply:
This width my screen. I writed width as in Figma.